### ISSUE TITLE
1 shouldNotBeGreaterThan 1

### ISSUE THREAD

**Issue by goreRatzete** [MarkusAmshove/Kluent#51]
I am running the following code: 
`1 shouldNotBeGreaterThan 1`

and get the following Error:
`java.lang.AssertionError: Expected 1 to not be greater than 1`

One is not greater than one, so the Test should pass, right?

The underlying assertion is actually checking if x is smaller than y. 
```
infix fun Int.`should not be greater than`(theOther: Int) = assertTrue("Expected $this to not be greater than $theOther", this < theOther)
```

**Comment by guenhter** [comment]
Would be fixed with #52

**Comment by MarkusAmshove** [comment]
I feel ashamed to introduce copy-paste bugs :-(

**Comment by MarkusAmshove** [comment]
I've released 1.26 with the fix from @guenhter , thank you two!
